gp-includes/things/note.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Things: GP_Note class
+ *
+ * @package GlotPress
+ * @subpackage Things
+ * @since 3.0.0
+ */
+
+/**
+ * Core class used to implement the notes system.
+ *
+ * @since 3.0.0
+ */
+class GP_Note extends GP_Thing {
+
```

Comment:
You need to include the `$table_basename`, `$field_names`, `$int_fields` and `$non_updatable_attributes` properties for the `thing`, otherwise it will error out on first install.

Reply:
done :-)